deployment/docker/Dockerfile

```diff
@@ -68,8 +68,8 @@ RUN python3 -m pip --no-cache-dir install Keras==2.1.6
 #
 # PyTorch
 #
-RUN python3 -m pip --no-cache-dir install torch==0.4.1
-RUN python3 -m pip install torchvision==0.2.1
+RUN python3 -m pip --no-cache-dir install torch==1.2.0
+RUN python3 -m pip install torchvision==0.4.0
 
```

Comment:
Suggest to use seperate PR to upgrade Dockfile

Reply:
Technically this PR won't work without this Dockerfile update. Maybe I should update the title.

Reply:
it's OK for me to change PR name.